src/main/java/io/confluent/connect/hdfs/avro/AvroHiveUtil.java

```diff
@@ -86,6 +86,7 @@ private Table constructAvroTable(
     Table table = newTable(database, tableName);
     table.setTableType(TableType.EXTERNAL_TABLE);
     table.getParameters().put("EXTERNAL", "TRUE");
+    // TODO not anymore, do we update the comment or do we need to update the path?
     // tableName is always the topic name
     String tablePath = hiveDirectoryName(url, config.getTopicsDirFromTopic(tableName), tableName);
```

Comment:
we need to change the code because the tableName passed here is already having "prefix-${topic}" in it and if we read cofig.getTopicsDirFromTopic(tableName) then it's documentation says one more replacement so it would be wrong if we replace tableName there instead of topic.

Top level directory to store the data ingested from Kafka. Supports ``${topic}`` in the value, which will be "
      replaced by the actual topic name. Supports ``${0}``, ..., ``${n}`` in ................."